### DISCUSSION TITLE
marker_quantification.generate_cell_table taking long time to complete

### DISCUSSION THREAD

**Discussion by genomelab** [angelolab/ark-analysis#1017] [General]
Is it common to have the generate_cell_table take several hours to complete (~3 hours for 450000 cells - on a pretty fast system).  I am using very large files (Akoya experiments) and there are 400K-2 million cells per image.  I only have one fov for the entire image (I segmented the entire image as a whole (both nuclear and membrane). Is there a way to break up the segmentation files by cell id and analyze the images in parallel?  I realize this is a one time analysis so it may not be high on the priority.  
Thank you for a great tool.

**Comment by srivarra** [comment]
Hi, thanks for checking out Ark!

One way that you can speed it up is setting `fast_extraction` to `True`, this will only extract the cell label, size, and the centroid, but not the other other expensive properties. If you don't need properties like 'area', 'eccentricity', 'major_axis_length', 'minor_axis_length', 'perimeter', 'convex_area', 'equivalent_diameter'.

And there are always some optimizations that we can do as well.
